apps/files/src/components/VirtualList.vue

```diff
@@ -296,7 +296,8 @@ export default defineComponent({
 
 			// Check if the content is smaller than the viewport, meaning no scrollbar
 			const targetRow = Math.ceil(this.dataSources.length / this.columnCount)
-			if (targetRow < this.rowCount) {
+			// The offset is needed to avoid some items not being visible but also not being scrolled to.
+			if (targetRow < this.rowCount - 8) {
```

Comment:
Sure it is not this? (8 is such a magic number)

```suggestion
			if (targetRow < (this.rowCount - ((this.bufferItems / this.columnCount) * 2)) {
```

Reply:
Yeah I hate this magic number. I just checked which value fits and 8 is the one. I'm not sure on what it is based, but I can try your calculation.

Reply:
So it's in list mode, but that doesn't work if the item is just one off screen. 8 is the minimum value that works here.

Reply:
Maybe we can tackle the scroll fix in a different PR then?

Reply:
Removed the commit for now.